python/graphscope/client/session.py

```diff
@@ -56,7 +56,7 @@
 from graphscope.framework.errors import K8sError
 from graphscope.framework.errors import LearningEngineInternalError
 from graphscope.framework.errors import check_argument
-from graphscope.framework.graph import Graph
+from graphscope.framework.graph import Graph, GraphDagNode
```

Comment:
let's consistently use DAG since it is an abbreviation. 
here, use `GraphDAGNode`